frontend/components/modal/ModalCommandPalette.vue

```diff
@@ -1,17 +1,22 @@
 <template>
+  <!-- TODO: transition / animation? Maybe do this in ModalBase? -->
   <ModalBase
     @closeModal="handleCloseModal"
     :isOpen="modalShouldClose == false ? modalIsOpen : false"
   >
+    <!-- SEARCH BOX -->
```

Comment:
For here, @mattburnett-repo, let's start using `MARK:` so that we get sections in the VS Code minimap 😊 I've started using them recently, and really find that it helps to organize things.